src/screens/HomePage/CriteriaExplanation/CriteriaExplanation.tsx

```diff
@@ -2,44 +2,85 @@ import React from 'react';
 import {
   Wrapper,
   Criterion,
-  Kicker,
+  MobileKicker,
+  DesktopKicker,
   CriterionHeader,
   CriterionDescription,
   CriteriaList,
+  ListHeader,
+  ListSubheader,
+  Content,
+  KickerWrapper,
 } from './CriteriaExplanation.style';
+import { BetaTag } from 'components/LocationPage/ChartsHolder.style';
 
 const CriteriaExplanation = () => {
   return (
     <Wrapper>
+      <ListHeader>How we determine risk levels</ListHeader>
+      <ListSubheader>Key indicators</ListSubheader>
       <CriteriaList>
         <Criterion>
-          <Kicker>Indicator 1</Kicker>
-          <CriterionHeader>Are COVID cases decreasing?</CriterionHeader>
-          <CriterionDescription>
-            Is the number of infections going down?{' '}
-          </CriterionDescription>
+          <KickerWrapper>
+            <MobileKicker>1</MobileKicker>
+            <DesktopKicker>Indicator 1</DesktopKicker>
+          </KickerWrapper>
```

Comment:
nit: Consider creating a unified Kicker component that will know how to display itself based on the screen size

```jsx
<Kicker number={1} title={Indicator 1} />
```

Reply:
done